recipe/meta.yaml

```
@@ -99,15 +99,13 @@ outputs:
         - pyarrow
         - python
         - scipy
-        - anndata       # [py>37]
-        - anndata <0.9  # [py<=37]
+        - anndata
```

```suggestion
        - anndata >=0.10.1
```

https://github.com/single-cell-data/TileDB-SOMA/blob/1.15.0rc3/apis/python/setup.py#L330

xref: https://github.com/TileDB-Inc/tiledbsoma-feedstock/issues/212